marklogic-data-hub/src/main/java/com/marklogic/hub/dhs/installer/command/AbstractInstallerCommand.java

```diff
@@ -93,7 +93,8 @@ protected ObjectNode canInstallDhs(){
             installedHubVersion = versions.getInstalledVersion();
         }
         catch (Exception e){
-            logger.info("Data Hub is not installed yet");
+            logger.error("Unable to determine the version of DHF: " + e.getMessage());
```

Comment:
Does this only happen when DH is not installed? Or could it be for other reasons? 

I think a better message is "Unable to determine version of existing Data Hub installation: " + e.getMessage(), unless we know that there is no existing DH installation.